onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_lib/NeuralNetworksWrapper.h

```diff
@@ -38,9 +36,7 @@ enum class Type {
   TENSOR_QUANT16_SYMM = ANEURALNETWORKS_TENSOR_QUANT16_SYMM,
   TENSOR_FLOAT16 = ANEURALNETWORKS_TENSOR_FLOAT16,
   TENSOR_BOOL8 = ANEURALNETWORKS_TENSOR_BOOL8,
-  FLOAT16 = ANEURALNETWORKS_FLOAT16,
   TENSOR_QUANT8_SYMM_PER_CHANNEL = ANEURALNETWORKS_TENSOR_QUANT8_SYMM_PER_CHANNEL,
-  TENSOR_QUANT16_ASYMM = ANEURALNETWORKS_TENSOR_QUANT16_ASYMM,
```

Comment:
why are we removing `TENSOR_QUANT16_ASYMM`/`FLOAT16` types here? 
is it to align with TF lite latest version

Reply:
yes, it matches this:
https://github.com/tensorflow/tensorflow/blob/d8ce9f9c301d021a69953134185ab728c1c248d3/tensorflow/lite/nnapi/NeuralNetworksTypes.h#L37-L51